h2o-py/tests/testdir_hstats/pyunit_h_categorical_columns.py

```diff
@@ -0,0 +1,38 @@
+from builtins import range
+import sys, os
+sys.path.insert(1, os.path.join("..",".."))
+import h2o
+from tests import pyunit_utils
+from h2o.estimators import H2OGradientBoostingEstimator
+import numpy as np
+import pandas as pd
+from h2o.exceptions import H2OServerError
+from pyunit_h_missing_values import prepare_data
```

Comment:
Can we also import it in pyunit_h_pubdev-8768_synthetic_data.py?

Reply:
I changed column x3 behavior and added x4 and x5 categorical columns. It could break a code for pyunit_h_pubdev-8768_synthetic_data.py.

Reply:
ok